<launch>

  <node pkg="learned_map" type="locTest" name="locTest" output="screen">

  <param name="loadMapName" value="/home/yh/mapModel/05.17/features_map_49068.vtk"/>
  <param name="loadTrajName" value="/home/yh/mapModel/05.18/Tianjin/icp_tianjin.txt"/>
  <param name="mapLength" value="10560"/>

  </node>
</launch>
